release-notes/elastic-cloud-serverless/index.md

```diff
@@ -9,6 +9,43 @@ products:
 # {{serverless-full}} changelog [elastic-cloud-serverless-changelog]
 Review the changes, fixes, and more to {{serverless-full}}.
 
+## July 22, 2025 [serverless-changelog-07222025]
+
+### Features and enhancements [serverless-changelog-07222025-features-enhancements]
+
+* Improves perceived performance for dashboard flyouts [#226052]({{kib-pull}}226052).
```

Comment:
```suggestion
* Improves perceived performance for dashboard flyouts [#226052]({{kib-pull}}226052)
```
Looking at the previous [Serverless release notes](https://www.elastic.co/docs/release-notes/cloud-serverless) I gather that we don't include a period at the end of each sentence, so I guess it should be removed from all of these entries.